airbyte-integrations/connectors/source-freshservice/source_freshservice/manifest.yaml

```
@@ -0,0 +1,190 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["{{ parameters.path_extractor }}"]
+  requester:
+    type: HttpRequester
+    url_base: "https://{{config['domain_name']}}/api/v2/"
+    http_method: "GET"
+    authenticator:
+      type: "BasicHttpAuthenticator"
+      username: "{{config['api_key']}}"
+      password: ""
```

```suggestion
      password: ""
    error_handler:
      response_filters:
        - http_codes: [403]
          action: IGNORE
```
add error handling for streams with no permissions.